style.css

```diff
@@ -1359,6 +1359,9 @@ canvas.nevysha {
     background-color: var(--background-fill-primary) !important;
 }
 
+.preview-block-wrapper {
```

Comment:
You can't do that because it create a huge empty gap under image preview
![image](https://github.com/Nevysha/Cozy-Nest/assets/122687716/a534c445-7de8-4b9f-ae0b-a36b27686c97)

Reply:
![image](https://github.com/Nevysha/Cozy-Nest/assets/167966/687c80f9-defa-496f-b6a6-9b6db4b0d083)

without it, `Show/Hide Generation Info` has chance out of screen, and can't be show up

Reply:
For sure this is an issue but I don't really want to fix a bug by adding another one

Reply:
if 120px is too more, can be 30px

---

or replace with txt2img_results padding-bottom: 30px;